.travis.yml

```diff
@@ -25,7 +29,7 @@ deploy:
 env:
   global:
   - PIPENV_IGNORE_VIRTUALENVS=1
-  - secure: ov66mTdCG+Gl+wLAjYdqR0HrKbgpQFSxSgCi0ihTmErzQl/vudZyPXdoSEQfFsWpKGI+UmhYklsvGySnYzyhTuUTrmovQ5C9tMMASWtJrwzAgun6PIli7ETjw5j273WTwIPy/HfrysqW6ab563VYbbuxn7ZANV86wN17TKp0qyaJPD4Kc6iFp8yKei7msXPy6Pc36AYz4vVjq1AZasFLIrRy5va4VNxtNUvK4peOdKcxDTmj4GMS8eF/Y55HuOzuH/B3hw0XqCgwk+u+4DCImpvQIhsfChgI71m70/scWwhiSaVY3WEmZBbJPgFLsoBm+div2+kG8zcNvOK2OEhQgvocxzgpTik5rvZX2SRCpuBxgbZEzUEbvNPArJVLwOBlbadIpuM8+sHTj/2A/4+AboVat10jjHrevTKlCo6CXe2QPXO474eZP0f5lOsGRQ9iSW52MRRCt8yOIALWwmn5DDrwt41ezn5+JDglxPxebqFLuvutG/7XMhhCuU91MYLxI/xhliRcYL9SiZXjnEz0p91YUNQQ2seJNvw/d65ypYdJDqS8kM8II7+lL/9l41hZ2En3LDSQ5diXPzEeNVp5cz/mn2rwlehTLmGbdb0Qy3WbD9eKoY1ADR+yDTE9l3t9UtRqkJ6Jgdig1YNkmdaFmrMfMHzTntY5He78b95zyk8=
+  - secure: qoMlJG2/e/ExVimSYtWbFxaePA7wSUgjsENaJPmAZKvisaSmbh3+BjlGc0128NViYGXMg2ljk2ofzST7/ktlJhnfewFrVHMctJsDppC0cmYXYQsXdGlFC59UDZXzp+jj4dRVSBiFUbIcznQDuGQ9z9rZs/dAHxPyVfgZoKbvuAiW+vNh+9veVvMfTFdSl6xNC4VBAk1BSWwRgYzYRyUFCXTmX+LbKN6SbyXpZXEwHoLCzKGePDK4SOHeFMSVhPryWtID1aCZYHHRm1ctFTjgvuAJ6s58Tlui5K9w4LFzlyhbWPE/T42cO1Azuqm+T/UZkD9k//rIj6yZb+JuxTNaIhosmB83MYzD/XhOEqCKoUjvYAHT1FwVpuml8/lm16EEloerag6yudAuVwerOQtPp3bcYANtKy78dLtVbu6AUDlpgjy3TELR60n02JDP2C5eu/3Bf2yLY3zOgshWluq6pTgX1PDaBgVL6PwDjnGtWI4Xpblt56rCYKvqHVqkQJzmTGP6epooiNEKaWy9tSo4aHRw82uzqOJY7Y3vr2eJ1vUtV5MBTSDnZyclRvEJA+qB6vHHSO7FRzTnzdvRmCO2VFOF+VfRGk8p331bBhotHIKxu5SwYxG97L8QuS4H9yni9zw039tCqluj2sEZ2xER/2AWCFtayty5ldz1SXIyY6U=
```

Comment:
I cannot decrypt, but I think it is a change like this:
```diff
-GH_TOKEN=xxxx
+GITHUB_TOKEN=xxx
```
We actually need both - `gen3git` needs the first one (perhaps it is even better to make gen3git support both)

Reply:
right 🤦‍♀